bundles/org.openhab.binding.miio/README.md

```diff
@@ -1131,6 +1143,45 @@ e.g. `smarthome:send actionCommand 'upd_timer["1498595904821", "on"]'` would ena
 | current          | Number  | Current                             |
 | temperature      | Number  | Temperature                         |
 
+### Vacuum 1C STYTJ01ZHM (<a name="dreame-vacuum-mc1808">dreame.vacuum.mc1808</a>) Channels
+
+| Channel          | Type    | Description                         |
+|------------------|---------|-------------------------------------|
+| BatteryLevel     | Number  | Battery-Battery Level               |
+| ChargingState    | Number  | Battery-Charging State              |
+| FirmwareRevision | String  | Device Information-Current Firmware Version |
+| Manufacturer     | String  | Device Information-Device Manufacturer |
+| Model            | String  | Device Information-Device Model     |
+| SerialNumber     | String  | Device Information-Device Serial Number |
+| Fault            | Number  | Robot Cleaner-Device Fault          |
+| Status           | Number  | Robot Cleaner-Status                |
+| BrushLeftTime    | String  | Main Cleaning Brush-Brush Left Time |
+| BrushLifeLevel   | Number  | Main Cleaning Brush-Brush Life Level |
+| FilterLifeLevel  | Number  | Filter-Filter Life Level            |
+| FilterLeftTime   | String  | Filter-Filter Left Time             |
+| BrushLeftTime1   | String  | Side Cleaning Brush-Brush Left Time |
+| BrushLifeLevel1  | Number  | Side Cleaning Brush-Brush Life Level |
+| WorkMode         | Number  | clean-workmode                      |
+| Area             | String  | clean-area                          |
+| Timer            | String  | clean-timer                         |
+| Mode             | Number  | clean-mode                          |
+| TotalCleanTime   | String  | clean-total time                    |
+| TotalCleanTimes  | String  | clean-total times                   |
```

Comment:
what is the difference between these fields? Can they be `Number:Time` instead?

Reply:
To be honest, I don't know. see the long comment below

Reply:
But, given the names, I expect that the one is the (clock) time of the cleaning and the other the amount of times the machine has cleaned. Similar counts are for the other Xiaomi vacuum cleaners available.

btw, it is still on my wish list #7276 to have the binding deal with `Number:Time` sort of notation. I expect it is not a huge effort, but I have not looked deeper into it what would be the detailed impact to the updating process and the possible impact in handling of the commands